mapie_v1/regression.py

```diff
@@ -105,7 +108,28 @@ def __init__(
         verbose: int = 0,
         random_state: Optional[Union[int, np.random.RandomState]] = None,
     ) -> None:
-        pass
+        
+        self.estimator = estimator
+        self.confidence_level = confidence_level
+        self.prefit = prefit
+        self.n_jobs = n_jobs
```

Comment:
- We don't need to use attributes for n_jobs, random_state, verbose, and conformity_score (for now, they are only used in __init__).
- We should chose to use either alpha or confidence_level, but not both.